vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/components/ExperimentalVariableContent.java

```diff
@@ -0,0 +1,19 @@
+package life.qbic.datamanager.views.projects.project.experiments.experiment.components;
+
+import java.util.List;
+
+/**
+ * @param name   the name of the variable
+ * @param unit   the unit of te variable levels
+ * @param levels the variable levels that are assigned to the variable
+ */
+public record ExperimentalVariableContent(String name, String unit, List<String> levels) {
```

Comment:
I thought about that. BUT:
The experimental variable content does not need to know about the Row Layout. The relation is the other way around. The row layout needs the content and not the content needs the layout.

Reply:
That's true, I think down the line we could introduce in a seperate PR a more convienient way to get the ExperimentalVariableContent from the VariableLevel to avoid this redundancy 🔆